sig-contributor-experience/annual-report-2023.md

```diff
@@ -3,17 +3,60 @@
+        - [Updates to Kubernetes Org Membership Requirements](https://groups.google.com/a/kubernetes.io/g/dev/c/X6zDzaDRYaU/m/nVgDcG1rAwAJ)
+        - [Onboarded SIG Etcd to Kubernetes GitHub Organisation & Prow](https://github.com/kubernetes/community/pull/7372) (major thanks to @cblecker, @mrbobbytables)
+
+    - Events:
+        - 3 successful Kubernetes contributors summits:
+            - [Europe, hosted in Amsterdam, Netherlands](https://www.kubernetes.dev/events/2023/kcseu/)
+            - [North America, hosted in Chicago, Illinois](https://www.kubernetes.dev/events/2023/kcsna/)
+            - [China, hosted in Shanghai](https://www.kubernetes.dev/events/2023/kcscn/) (special thanks to @pacoxu for [leading the event](https://github.com/kubernetes/community/issues/7510))
+
+    - Mentoring:
+        - 1 successful mentoring cohort:
+            - [Kustomize maintainer training cohort](https://groups.google.com/a/kubernetes.io/g/dev/c/M5OphEVsv5o/m/zc6G4H15AAAJ)
+
+    - Elections:
+        - [Kubernetes Steering Committee Elections, for the 2023 cycle](https://groups.google.com/a/kubernetes.io/g/dev/c/bZpmcKA_900/m/EnlPmiDBAQAJ)
+        - facilitated [Kubernetes Code of Conduct Committee Election, for the 2023 cycle](https://groups.google.com/a/kubernetes.io/g/dev/c/YfyhGdTr3iA/m/AxT0OtIjBAAJ)
+
+    - Contributor-comms:
+        - got access to Linkedin
+        - started process to get access to kubernetes twitter account
+        - LWKD (Last Week in Kubernetes Development) group became part of contributor-comms, onboarded 3 new dedicated maintainers [1](https://github.com/kubernetes/org/pull/4519) [2](https://github.com/kubernetes/org/pull/4818)
+        - SIG spotlight blogs
+            - how many?
+            - how many by new contributors?
+        - new subproject leadership roles?
+
+    - Slack moderation:
+        - [Updates to Kubernetes core contributor slack channels](https://groups.google.com/a/kubernetes.io/g/dev/c/E1SeJvWQgzQ/m/EkN_w2E1AQAJ) - #kubernetes-new-contributors, #kubernetes-org-members
+
 
-<!--
-   Some example items that might be worth highlighting:
-   - Major KEP advancement
-   - Important initiatives that aren't tracked via KEPs
-   - Paying down significant tech debt
-   - Governance and leadership changes
--->
 
 2. Are there any areas and/or subprojects that your group needs help with (e.g. fewer than 2 active OWNERS)?
 
+
+    - Mentoring subproject needs a dedicated leadership, responsible for running both (i) group mentoring cohorts for Kubernetes SIGs & WGs, (ii) external mentoring programs (LFX Mentorship, Google Summer of Code, Google Summer of Docs, Outreachy, etc)
+
+
+    - Elections subproject is need of a lead, who will own & maintain election software, pick election officers, and  help groups in the Kubernetes project, who want to use the election software.
+
```

Comment:
Follow up from the `April 21, 2024` ContribEx bi-weekly meeting:

Hello @jberkus, ping for collecting specifics for the help needed for mentoring & elections subproject. Thanks!